### ISSUE TITLE
"Scroll to bottom/top" button for issue comments

### ISSUE THREAD

**Issue by StanNL** [k0shk0sh/FastHub#556]
---

 - FastHub Version: 2.5.1
 - Android Version: 7.1.2
 - Phone Model: Samsung kminilte

As you probably already know by now, I participate in some extremly long issue conversations with hundreds of comments, so it would be nice to have a button that allows me to scroll all the way to the bottom or back to the top of the conversation.

**Comment by k0shk0sh** [comment]
@StanNL isn't the fastScroller is enough? & clicking on tabs if applicable to take u top is enough? 

tbh with you,  I couldn't find a place or could imagine the design that would fit this, as its unusual behaviour for android. so using the FastScroller I guess is enough for this. 
I hope you understand :)

**Comment by StanNL** [comment]
@k0shk0sh Yeah I also thought about that, but I also wouldn't know where to put it. 

In that case, there is something else that could be changed which is related to this. In version 2.5.3, the app is able to load more than 100 comments, but it only does that once you've scrolled through the first 100. So when you use the FastScroller, you have to stop after 100 comments and wait until it has loaded, and then you can continue using the scroller, until you've reached the 200th comment, and you have to stop again and wait, etc. etc. So in my opinion, it would be better to load the 1st-100th comment first and then just display them, like you're doing right now, but after that's done, you could also start loading the 101st-200th and 201st-300th comment so that you can just use the FastScroller all the way to the bottom.

In short: it would be nice if you could also _preload_ the other comments that are not currently shown. I hope I was clear enough.

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
